Src/Core/Equivalency/FieldSelectedMemberInfo.cs

```diff
@@ -30,5 +31,15 @@ public override Type MemberType
         {
             get { return fieldInfo.FieldType; }
         }
+
+        public override CSharpAccessModifier GetAccessModifier
```

Comment:
Indeed this looks like it would point the user into the right direction.
- Wouldn't `WhichHasSetterModifier` be better ? Or is this the followed convention, with the `Which` ?
- What would the negated version of this be named ? (This is the one I personally need)
- So no more public property on SelectedMemberInfo for the access modifier.